### PR TITLE
Fixes #20983: INFO should be NOTE

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/20983

### CODE REVIEW

**Comment by fanf** [comment]
PR updated with a new commit

File: src/reference/modules/usage/pages/advanced_node_management.adoc [hunk 1]
```diff
@@ -452,6 +452,7 @@ image::node-lifecycle-settings.png[]
 In the future, these states will be configurable on a per node basis at acceptation, and the
 lifecycle states list will be configurable by users.
 
+[[disable_listen_port_5309]]
```
Comment (VinceMacBuche) [inline src/reference/modules/usage/pages/advanced_node_management.adoc:455]: is it not a problem that there is no space between this line and the one below ?

Comment (fanf) [reply src/reference/modules/usage/pages/advanced_node_management.adoc:455]: no, for reference it's done like that elsewhere (and I see that the two of us are very fluent in doc :)) )

Comment (VinceMacBuche) [reply src/reference/modules/usage/pages/advanced_node_management.adoc:455]: totally fluent, and i would have done like you, copy from other file :)

**Review by VinceMacBuche — APPROVED** [review]

**Pull request merged**